src/frontend/src/components/InvalidInput/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+interface Props {
+  message: any;
+  condition: boolean;
+}
+
+export default class InvalidInput extends React.Component<Props> {
+  render() {
+    return this.props.condition ? (
+      <div role="alert" className="black-70 w-100">
```

Comment:
The `role=alert` container should be in the dom before the message is inserted in order for screen readers to announce it: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19

Reply:
Ok. I changed it to always render the `div` for the component, but that also means the page always renders the stack of empty `alert div`s before populating any of them. The layout looks fine, but should we avoid this and still wrap all of these `<p>` elements in a single container?

Reply:
I think a single container is ideal.